src/Application/Actions/Hooks/StripePaymentsUpdate.php

```diff
@@ -97,21 +97,27 @@ protected function action(Request $request, Response $response, array $args): Re
         $type = $event->type;
         $this->logger->info(sprintf('Received Stripe account event type "%s"', $type));
 
-        switch ($type) {
-            case Event::CHARGE_DISPUTE_CLOSED:
-                return $this->handleChargeDisputeClosed($event, $response);
-            case Event::CHARGE_REFUNDED:
-                return $this->handleChargeRefunded($event, $response);
-            case Event::CHARGE_SUCCEEDED:
-                return $this->handleChargeSucceeded($event, $response);
-            case Event::PAYMENT_INTENT_CANCELED:
-                return $this->handlePaymentIntentCancelled($event, $response);
-            case Event::CUSTOMER_CASH_BALANCE_TRANSACTION_CREATED:
-                return $this->handleCashBalanceUpdate($event, $response);
-            default:
-                $this->logger->warning(sprintf('Unsupported event type "%s"', $type));
+        return match ($type) {
```

Comment:
I thought match was slightly simpler and more robust than switch, although its marginal in this case.